Gruntfile.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(grunt) {
+  grunt.initConfig({
+    jshint: {
+      all: ['Gruntfile.js', 'index.js']
```

Comment:
@emmettu good point. As for JSXHint its now depracated, so I'll try looking into ESLint.